<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cerrarAplicativo">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>520</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar aplicativo</string>
    </property>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>581</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CORRELACIONES SPEARMANR Y PEARSON</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Seleccione el archivo</string>
    </property>
    <widget class="QPushButton" name="cargarArchivo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccionar archivo ...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textArchivo">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>541</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Procesar correlaci√≥n</string>
    </property>
    <widget class="QPushButton" name="procesarArchivo">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Procesar archivo ...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="resultadoCorrelacion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>501</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
